recettes/sucrees/galette_des_rois.md

```diff
@@ -0,0 +1,31 @@
+# Galette des rois
```

Comment:
"a la frangipane" peut etre?

Reply:
est-ce que je devrais changer aussi le nom du fichier, ou juste le titre de la recette? what do you think?

Reply:
peut etre juste le titre, on pourra toujours changer le nom du fichier si il y a une autre galette des rois un jour